LZW/LZW/LZWDecoder.cs

```diff
@@ -0,0 +1,125 @@
+namespace LZW;
+
+/// <summary>
+/// Class that represents LZW-decoder.
+/// </summary>
+public static partial class LZWDecoder
+{
+    private const int StartDictionarySize = 256;
+
+    private const int StartMaximumAmountOfCodesNumber = 512;
+
+    /// <summary>
+    /// Decodes input array of LZW-encoded bytes.
+    /// </summary>
+    /// <param name="input">Input array of LZW-encoded bytes.</param>
+    /// <returns>Array of LZW-decoded bytes.</returns>
+    public static byte[] Decode(byte[]? input)
+    {
+        if (input == null)
+        {
+            throw new ArgumentNullException("Input bytes stream can't be null!");
```

Comment:
```suggestion
            throw new ArgumentNullException(nameof(input));
```
Что кто-то-там не может быть null, понятно из типа исключения. Указать параметр, который не понравился методу, нелишне (если бы параметров было больше одного, то тем более). nameof нужен, чтобы при автоматическом переименовании параметра (при рефакторинге) в исключении он тоже переименовался.